client-app/pages/account/quotes.vue

```diff
@@ -243,7 +245,12 @@ async function changePage(newPage: number): Promise<void> {
 function goToQuoteDetails(payload: { id: string; status?: string }): void {
   const pathName: string = payload.status === "Draft" ? "EditQuote" : "ViewQuote";
   const quoteRoute = router.resolve({ name: pathName, params: { quoteId: payload.id } });
-  window.open(quoteRoute.fullPath, "_blank")!.focus();
+
+  const target = themeContext.value.settings.show_details_in_separate_tab
+    ? getLinkTarget(themeContext.value.settings.show_details_in_separate_tab)
```

Comment:
seems that `const target = getLinkTarget(themeContext.value.settings.show_details_in_separate_tab);`
does the same thing, so ternary operator is a bit redundant here.